internal/webserver/webserver.go

```diff
@@ -502,3 +508,29 @@ func (n *kubeFilter) removingHopByHopHeaders(request *http.Request) {
 
 	request.Header.Del(connectionHeaderName)
 }
+
+func (n *kubeFilter) BearerToken() string {
+	if !n.bearerTokenExpirationTime.After(time.Now()) {
```

Comment:
```suggestion
	if time.Now().After(!n.bearerTokenExpirationTime) {
```

Isn't this more readable? 🤔